packages/hub/src/components/GlobalSearch/SearchResultGroup.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from "react";
+import { graphql, useFragment } from "react-relay";
+
+import { SearchResultGroup$key } from "@/__generated__/SearchResultGroup.graphql";
+import { SearchResultBox } from "./SearchResultBox";
+import { SearchResultComponent, useEdgeFragment } from "./SearchResult";
+import { Snippet } from "./Snippet";
+import { SearchResultTitle } from "./SearchResultTItle";
+
+export const SearchResultGroup: SearchResultComponent<
+  SearchResultGroup$key
+> = ({ fragment, edgeFragment }) => {
+  const edge = useEdgeFragment(edgeFragment);
+
+  // Unused, because `SearchEdge.slugSnippet` is better than `Group.slug`.
```

Comment:
In the earlier version, I used `Group.slug`, but after I implemented `slugSnippet` I realized that there's nothing that this component (or `SearchResultUser` component) need from the Group or User objects themselves.

But since `SearchResultComponent` is a generic, and since `<SearchResult>` shouldn't care much about which components require data from the found object and which don't, I think it's fine if we keep this for now.